packages/invoices/src/OutboundShipment/DetailView/SidePanel.tsx

```diff
@@ -114,11 +114,29 @@ export const SidePanel: FC<SidePanelProps> = ({ draft }) => {
   return (
     <DetailPanelPortal
       Actions={
-        <DetailPanelAction
-          icon={<CopyIcon />}
-          titleKey="link.copy-to-clipboard"
-          onClick={copyToClipboard}
-        />
+        <>
+          {!process.env['NODE_ENV'] ||
+            (process.env['NODE_ENV'] === 'development' && (
+              <DetailPanelAction
+                icon={<CopyIcon />}
+                titleKey="dev.log-draft"
+                onClick={() => {
+                  console.table(draft);
+                  draft.items.forEach(item => {
+                    console.table(item);
+                    Object.values(item.batches).forEach(batch => {
+                      console.table(batch);
+                    });
+                  });
+                }}
+              />
+            ))}
```

Comment:
Can also remove this if you'd like. Also, does this dev env work? 🤷 

This was the first time i've really wanted some debugging (need to set up the source maps for debugging nicely, I think) - basically just a button to log the draft, which was pretty helpful for me at the time.